src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -19,16 +29,32 @@ export interface BreadcrumbProps extends React.HTMLAttributes<HTMLDivElement> {
  *
  * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-breadcrumb--overview)
  */
-const Breadcrumb = ({ className, items = [], ...props }: BreadcrumbProps) => {
+const Breadcrumb = ({ className, items = [], truncateIndex, truncateLength = 10, ...props }: BreadcrumbProps) => {
   const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Breadcrumb');
 
   return (
     <nav aria-label="Breadcrumb" className={classnames(baseClassName, className)} {...commonProps} {...props}>
-      <ol>
-        {items.map((item: BreadcrumbItemProps, index: number) => (
-          <BreadcrumbItem href={item.href} label={item.label} isCurrent={items.length - 1 === index} key={item.label} />
-        ))}
-      </ol>
+      {/* This is not visible when in desktop breakpoint */}
+      <SSRMediaQuery.Media lessThan="md">
+        <button onClick={() => (window.location.href = items[1].href ? items[1].href : '')} className="back-button">
```

Comment:
can we move this function out and name it something like `goBack`

Reply:
in the figma here, I see spacer medium top and bottom, and no padding/spacer on the left and right:
https://www.figma.com/design/hxqgsE26wM7hII0WaUaDfF/RW---TIMED-Lot-Details-(PDP)?node-id=189-5586&m=dev

Reply:
this is not a Remixy way of doing this.  Can we please render this as a Link instead of a button and allow users to pass the backLink element as a custom element so I can pass a `<Link` from Remix?